psync/src/main/cpp/psync-wrapper.cpp

```diff
@@ -375,10 +376,19 @@ JNIEXPORT jboolean JNICALL Java_net_named_1data_jni_psync_PSync_00024Consumer_ad
   (JNIEnv *env, jobject, jobject handle, jstring prefix)
 {
   ConsumerWrapper* consumerWrapper = (ConsumerWrapper*) env->GetDirectBufferAddress(handle);
-  ALOG("%s", "Trying to add subscription for consumer");
   return consumerWrapper->consumer->addSubscription(ndn::Name(env->GetStringUTFChars(prefix, nullptr)));
 }
 
+JNIEXPORT void JNICALL Java_net_named_1data_jni_psync_PSync_00024Consumer_stop
+  (JNIEnv *env, jobject, jobject handle)
+{
+    ConsumerWrapper* consumerWrapper = (ConsumerWrapper*) env->GetDirectBufferAddress(handle);
+    consumerWrapper->consumer->stop();
+    delete consumerWrapper;
+    ALOG("%s", "Stopping consumer");
```

Comment:
Maybe Stopped consumer.

Reply:
This was just forgotten debug code. I'll remove it.